src/celeritas/track/SimData.hh

```diff
@@ -75,7 +75,7 @@ struct SimParamsData
     //// METHODS ////
 
     //! Whether the data are assigned
-    explicit CELER_FUNCTION operator bool() const { return !looping.empty(); }
+    explicit CELER_FUNCTION operator bool() const { return true; }
```

Comment:
Looks like we also need to relax the requirement that `num_looping_steps` is allocated in the state `operator bool()`.

Reply:
Yeah, this is a rebase error 😅 will fix in a minute, thanks!